apps/labs/posts/scikit-lego-narwhals.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'How Narwhals and scikit-lego came together to achieve dataframe-agnosticism'
+authors: [marco-gorelli]
+published: May 26, 2024
+description: 'And how your library can become dataframe-agnostic too'
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/scikit-lego-narwhals/scikit_lego_narwhals_handshake.png
+  alt: 'Scikit-lego and Narwhals logos shaking hands'
+hero:
+  imageSrc: /posts/scikit-lego-narwhals/hero.png
+  imageAlt: 'Scikit-lego and Narwhals logos'
+---
+
+[Scikit-lego](https://github.com/koaning/scikit-lego) is a Python project from the [scikit-learn](https://scikit-learn.org/stable/) ecosystem that contributes extra estimators for machine-learning pipelines.
+It's a relatively popular project with over 1000 Github stars and 20,000 downloads a month.
```

Comment:
If this is a new paragraph there should be a blank new line between the two.

Reply:
it's the same one 😉